py/server/deephaven/stream/kafka/consumer.py

```diff
@@ -388,3 +389,28 @@ def simple_spec(col_name: str, data_type: DType = None) -> KeyValueSpec:
         )
     except Exception as e:
         raise DHError(e, "failed to create a Kafka key/value spec") from e
+
+
+def raw_spec(col_name: str, data_type: str, deserializer: str) -> KeyValueSpec:
+    """Creates a raw spec with explicit expected data_type and kafka deserializer.
+
+    Args:
+        col_name (str): the Deephaven column name
+        data_type (str): the column data type class
```

Comment:
Is there a reason this is a string and not a `DType`?  I think I understand the reasoning, but I want to confirm.